prowler/providers/aws/services/cognito/cognito_user_pool_advanced_security_block_sign_in_compromised_credentials/cognito_user_pool_advanced_security_block_sign_in_compromised_credentials.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "aws",
+  "CheckID": "cognito_user_pool_advanced_security_block_sign_in_compromised_credentials",
+  "CheckTitle": "Ensure cognito user pools has advanced security enabled with full-function to block sign-in by users with suspected compromised credentials",
+  "CheckType": [],
+  "ServiceName": "cognito",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:aws:cognito-idp:region:account:userpool/userpool-id",
+  "Severity": "medium",
+  "ResourceType": "AwsCognitoUserPool",
+  "Description": "Checks whether advanced security features are enabled for an Amazon Cognito User Pool to block sign-in by users with suspected compromised credentials.",
+  "Risk": "If advanced security features are not enabled for an Amazon Cognito User Pool, the user pool will not be able to block sign-in by users with suspected compromised credentials.",
+  "RelatedUrl": "https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-settings-advanced-security.html",
+  "Remediation": {
+    "Code": {
+      "CLI": "",
+      "NativeIaC": "",
+      "Other": "",
+      "Terraform": ""
+    },
+    "Recommendation": {
+      "Text": "To enable advanced security features for an Amazon Cognito User Pool, follow the steps in the Amazon Cognito Developer Guide.",
+      "Url": "https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-settings-advanced-security.html"
+    }
```

Comment:
Review this again with the new check name.

Reply:
Done